site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -59,37 +60,90 @@ investigate, but you want to pause only when some other condition is true.
 
 To set a conditional line-of-code breakpoint:
 
-1.  Click the **Sources** tab.
+1.  Open the **Sources** tab.
 2.  Open the file containing the line of code you want to break on.
 3.  Go to the line of code.
 4.  To the left of the line of code is the line number column. Right-click it.
 5.  Select **Add conditional breakpoint**. A dialog displays underneath the line of code.
 6.  Enter your condition in the dialog.
-7.  Press Enter to activate the breakpoint. An orange icon appears on top of the line number column.
+7.  Press <kbd>Enter</kbd> to activate the breakpoint. An orange icon with a question mark appears on top of the line number column.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/wbb7VQOAG1GOHvBqqjdt.png", alt="A conditional line-of-code breakpoint.", width="800", height="579" %}
```

Comment:
The walkthrough should include an example of condition or usecase to help beginner understand better. 
Probably add some example in step 6, or explain your screenshot on why you want to do that in real-life.

Currently the code example only show in image, which is not accessible.